### ISSUE TITLE
Add dedicated element pickers for "section"-tag to the DSL

### ISSUE THREAD

**Issue by skrapeit** [skrapeit/skrape.it#25]
currently it is possible to to pick section-tag using

```
...
expect {
    element("section") {
        text() toBe "I'm the inner text"
     }
}
```

make it possible to have a more convenient and type safe way of picking elements.

e.g.:

```
...
expect {
    section {
        text() toBe "I'm the inner text"
     }
}
```

**Comment by gemasr** [comment]
I'll pick this one!

**Comment by skrapeit** [comment]
FYI: simular issue has been done yesterday: https://github.com/skrapeit/skrape.it/issues/47
currently its a bit like maintaining the same stuff twice in the `DocExtractors.kt` and the `ResultExtractors.kt`. if you have a nice idea to unify this would be awesome but just adding the section picker like it has been done for spans for example would be ok as well :)

**Comment by skrapeit** [comment]
resolved with commit 4dcb807
